test/integration/patients/patient/dashboard.js

```diff
@@ -405,13 +406,16 @@ context('patient dashboard page', function() {
       .should('contain', `patient-action/${ testAction.id }/form/${ testForm.id }`);
   });
 
-  specify.only('add action and flow', function() {
+  specify('add action and flow', function() {
     const currentClinican = getCurrentClinician({
       relationships: {
         role: getRelationship(roleEmployee),
       },
     });
 
+    const testProgramActionIds = [uuid(), uuid(), uuid(), uuid(), uuid(), uuid()];
+    const testProgramFlowIds = [uuid(), uuid(), uuid(), uuid(), uuid(), uuid()];
```

Comment:
I needed to set these all at the top. And then use them throughout.

Otherwise using e.g. `testProgramActions[0].id` or `testProgramFlows[0].id` in `testPrograms = []` would cause errors. Since we'd be trying to use data/variables that weren't instantiated yet.

Reply:
I think if you flipped it and made programIds with uuid then you could define the flows/actions first and use the ids generated from there for relations on the program.
You could also use `_.times` with `uuid`

Reply:
Think that works well 👍 , and `_.times` is also much better